### ISSUE TITLE
Lack of documentation

### ISSUE THREAD

**Issue by wilson-csdshfl** [sirupsen/zk#16]
I think it would be great to make documentation as to how to use "zk". The readme.md is not sufficient enough, as I still get trouble to start using this. Yes, I can just read the code, but i will be grateful if you could provide practical use of it. Thanks.

**Comment by riodelphino** [comment]
It seems to nice TUI tool for zk, but I can't start using too.
zk says " cp: index.db: No such file or directory"

**Comment by JulianNymark** [comment]
yeah, the project does have a disclaimer, but it's pretty neat as is, so once i've gone through it a bit more & understand the pieces i can look into making some improved docs 🙌.

in short, a few errors i encountered + solutions/workarounds:

> cp: index.db: No such file or directory

it complained about this due to the directory that it tries to copy the sqlite db  _into_ doesn't exist (`backup/`), at least that was the case on my end, so in my $ZK_PATH I ran `mkdir backup` and the error went away.

> flickering failures in the fzf preview window (nothing shows, and sometimes an error about index.db being locked)

This seemed to be resolved by adding a microscopic (10ms or so) sleep before fzf edits would run the the `zk-fts-search` script. I think it was running the new fzf command faster than the lock had been released from the previous run. (could be a myriad of reasons lower down, but the sleep seems like a nice hack for me currently, I don't notice 10ms). This could also be a mac specific issue.
```shell
fzf
...
--bind "change:reload(sleep 0.01s; zk-fts-search --backup '{q}')" \
...
```
